### ISSUE TITLE
Sbt doesn't compile when PlayAkkaHttp2Support is enabled

### ISSUE THREAD

**Issue by amarjanica** [playframework/playframework#7517]
Sbt play fails to enter interactive mode when **PlayAkkaHttp2Support** plugin is enabled. I added a simple project which can reproduce the bug, and below is stacktrace.

### Play Version (2.5.x / etc)
2.6.0

### API (Scala / Java / Neither / Both)
Java

### Operating System (Ubuntu 15.10 / MacOS 10.10 / Windows 10)
Ubuntu 16.04.2 LTS

### JDK (Oracle 1.8.0_72, OpenJDK 1.8.x, Azul Zing)
1.8.0_101

### Library Dependencies
None

### Expected Behavior
Enter sbt interactive mode.

### Actual Behavior
Sbt fails with 
```
Java HotSpot(TM) 64-Bit Server VM warning: ignoring option MaxPermSize=256M; support was removed in 8.0
[info] Loading global plugins from /home/ana/.sbt/0.13/plugins
[info] Loading project definition from /home/ana/work/code/play/playhttp2-bug/project
java.lang.NoSuchMethodError: com.typesafe.sbt.packager.archetypes.JavaAppPackaging$autoImport$.bashScriptExtraDefines()Lsbt/TaskKey;
	at com.lightbend.sbt.javaagent.JavaAgentPackaging$.projectSettings(JavaAgentPackaging.scala:26)
	at sbt.Load$$anonfun$resolveProject$1$$anonfun$autoPluginSettings$1$1$$anonfun$apply$34.apply(Load.scala:721)
	at sbt.Load$$anonfun$resolveProject$1$$anonfun$autoPluginSettings$1$1$$anonfun$apply$34.apply(Load.scala:721)
	at scala.collection.TraversableLike$$anonfun$flatMap$1.apply(TraversableLike.scala:251)
	at scala.collection.TraversableLike$$anonfun$flatMap$1.apply(TraversableLike.scala:251)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at scala.collection.TraversableLike$class.flatMap(TraversableLike.scala:251)
	at scala.collection.AbstractTraversable.flatMap(Traversable.scala:105)
	at sbt.Load$$anonfun$resolveProject$1$$anonfun$autoPluginSettings$1$1.apply(Load.scala:721)
	at sbt.Load$$anonfun$resolveProject$1$$anonfun$autoPluginSettings$1$1.apply(Load.scala:721)
	at sbt.Load$.timed(Load.scala:1025)
	at sbt.Load$$anonfun$resolveProject$1.autoPluginSettings$1(Load.scala:720)
	at sbt.Load$$anonfun$resolveProject$1.sbt$Load$$anonfun$$expandSettings$1(Load.scala:739)
	at sbt.Load$$anonfun$resolveProject$1$$anonfun$sbt$Load$$anonfun$$expandSettings$1$2.apply(Load.scala:740)
	at sbt.Load$$anonfun$resolveProject$1$$anonfun$sbt$Load$$anonfun$$expandSettings$1$2.apply(Load.scala:740)
	at scala.collection.IndexedSeqOptimized$class.foldl(IndexedSeqOptimized.scala:51)
	at scala.collection.IndexedSeqOptimized$class.foldLeft(IndexedSeqOptimized.scala:60)
	at scala.collection.mutable.WrappedArray.foldLeft(WrappedArray.scala:34)
	at scala.collection.TraversableOnce$class.$div$colon(TraversableOnce.scala:138)
	at scala.collection.AbstractTraversable.$div$colon(Traversable.scala:105)
	at sbt.Load$$anonfun$resolveProject$1.sbt$Load$$anonfun$$expandSettings$1(Load.scala:740)
	at sbt.Load$$anonfun$resolveProject$1$$anonfun$56.apply(Load.scala:743)
	at sbt.Load$$anonfun$resolveProject$1$$anonfun$56.apply(Load.scala:743)
	at sbt.Load$.timed(Load.scala:1025)
	at sbt.Load$$anonfun$resolveProject$1.apply(Load.scala:742)
	at sbt.Load$$anonfun$resolveProject$1.apply(Load.scala:703)
	at sbt.Load$.timed(Load.scala:1025)
	at sbt.Load$.resolveProject(Load.scala:703)
	at sbt.Load$.sbt$Load$$finalizeProject$1(Load.scala:603)
	at sbt.Load$$anonfun$51.apply(Load.scala:637)
	at sbt.Load$$anonfun$51.apply(Load.scala:637)
	at sbt.Load$.timed(Load.scala:1025)
	at sbt.Load$.sbt$Load$$loadTransitive(Load.scala:636)
	at sbt.Load$$anonfun$loadUnit$1.sbt$Load$$anonfun$$loadProjects$1(Load.scala:482)
	at sbt.Load$$anonfun$loadUnit$1$$anonfun$40.apply(Load.scala:485)
	at sbt.Load$$anonfun$loadUnit$1$$anonfun$40.apply(Load.scala:485)
	at sbt.Load$.timed(Load.scala:1025)
	at sbt.Load$$anonfun$loadUnit$1.apply(Load.scala:485)
	at sbt.Load$$anonfun$loadUnit$1.apply(Load.scala:459)
	at sbt.Load$.timed(Load.scala:1025)
	at sbt.Load$.loadUnit(Load.scala:459)
	at sbt.Load$$anonfun$25$$anonfun$apply$14.apply(Load.scala:311)
	at sbt.Load$$anonfun$25$$anonfun$apply$14.apply(Load.scala:310)
	at sbt.BuildLoader$$anonfun$componentLoader$1$$anonfun$apply$4$$anonfun$apply$5$$anonfun$apply$6.apply(BuildLoader.scala:91)
	at sbt.BuildLoader$$anonfun$componentLoader$1$$anonfun$apply$4$$anonfun$apply$5$$anonfun$apply$6.apply(BuildLoader.scala:90)
	at sbt.BuildLoader.apply(BuildLoader.scala:140)
	at sbt.Load$.loadAll(Load.scala:365)
	at sbt.Load$.loadURI(Load.scala:320)
	at sbt.Load$.load(Load.scala:316)
	at sbt.Load$.load(Load.scala:305)
	at sbt.Load$$anonfun$4.apply(Load.scala:146)
	at sbt.Load$$anonfun$4.apply(Load.scala:146)
	at sbt.Load$.timed(Load.scala:1025)
	at sbt.Load$.apply(Load.scala:146)
	at sbt.Load$.defaultLoad(Load.scala:39)
	at sbt.BuiltinCommands$.liftedTree1$1(Main.scala:548)
	at sbt.BuiltinCommands$.doLoadProject(Main.scala:548)
	at sbt.BuiltinCommands$$anonfun$loadProjectImpl$2.apply(Main.scala:540)
	at sbt.BuiltinCommands$$anonfun$loadProjectImpl$2.apply(Main.scala:540)
	at sbt.Command$$anonfun$applyEffect$1$$anonfun$apply$2.apply(Command.scala:59)
	at sbt.Command$$anonfun$applyEffect$1$$anonfun$apply$2.apply(Command.scala:59)
	at sbt.Command$$anonfun$applyEffect$2$$anonfun$apply$3.apply(Command.scala:61)
	at sbt.Command$$anonfun$applyEffect$2$$anonfun$apply$3.apply(Command.scala:61)
	at sbt.Command$.process(Command.scala:93)
	at sbt.MainLoop$$anonfun$1$$anonfun$apply$1.apply(MainLoop.scala:96)
	at sbt.MainLoop$$anonfun$1$$anonfun$apply$1.apply(MainLoop.scala:96)
	at sbt.State$$anon$1.doX$1(State.scala:183)
	at sbt.State$$anon$1.process(State.scala:190)
	at sbt.MainLoop$$anonfun$1.apply(MainLoop.scala:96)
	at sbt.MainLoop$$anonfun$1.apply(MainLoop.scala:96)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
	at sbt.MainLoop$.next(MainLoop.scala:96)
	at sbt.MainLoop$.run(MainLoop.scala:89)
	at sbt.MainLoop$$anonfun$runWithNewLog$1.apply(MainLoop.scala:68)
	at sbt.MainLoop$$anonfun$runWithNewLog$1.apply(MainLoop.scala:63)
	at sbt.Using.apply(Using.scala:24)
	at sbt.MainLoop$.runWithNewLog(MainLoop.scala:63)
	at sbt.MainLoop$.runAndClearLast(MainLoop.scala:46)
	at sbt.MainLoop$.runLoggedLoop(MainLoop.scala:30)
	at sbt.MainLoop$.runLogged(MainLoop.scala:22)
	at sbt.StandardMain$.runManaged(Main.scala:109)
	at sbt.xMain.run(Main.scala:38)
	at xsbt.boot.Launch$$anonfun$run$1.apply(Launch.scala:109)
	at xsbt.boot.Launch$.withContextLoader(Launch.scala:128)
	at xsbt.boot.Launch$.run(Launch.scala:109)
	at xsbt.boot.Launch$$anonfun$apply$1.apply(Launch.scala:35)
	at xsbt.boot.Launch$.launch(Launch.scala:117)
	at xsbt.boot.Launch$.apply(Launch.scala:18)
	at xsbt.boot.Boot$.runImpl(Boot.scala:41)
	at xsbt.boot.Boot$.main(Boot.scala:17)
	at xsbt.boot.Boot.main(Boot.scala)
[error] java.lang.NoSuchMethodError: com.typesafe.sbt.
packager.archetypes.JavaAppPackaging$autoImport$.bashScriptExtraDefines()Lsbt/TaskKey;
```

### Reproducible Test Case
I added a play project with reproducible bug.
[playhttp2-bug.zip](https://github.com/playframework/playframework/files/1108033/playhttp2-bug.zip)

**Comment by gmethvin** [comment]
This seems to be related to https://github.com/sbt/sbt-javaagent/commit/1e79a1822eb051165c3b5a154da52246b55079ae

I think we need to update to sbt-javaagent 0.1.3. As a workaround try adding this to `plugins.sbt`:

```
addSbtPlugin("com.lightbend.sbt" % "sbt-javaagent" % "0.1.3")
```

**Comment by amarjanica** [comment]
Workaround works! Thanks

**Comment by richdougherty** [comment]
Hi @amarjanica, this issue is now addressed in Play 2.6.1 if you want to try it out.
